app/concerns/twitter_concern.rb

```diff
@@ -36,7 +36,7 @@ def twitter_oauth_token_secret
   end
 
   def twitter
-    Twitter::REST::Client.new do |config|
+    @twitter ||= Twitter::REST::Client.new do |config|
```

Comment:
I wanted to mock a method on `twitter` in my spec and generating a new one every time was not allowing that. I could use `any_instance_of(Twitter::REST::Client)` but I didn't see a reason not to cache this.